projects/storefrontapp-e2e-cypress/cypress.config.ts

```diff
@@ -9,6 +9,7 @@ import { defineConfig } from 'cypress';
 export default defineConfig({
   defaultCommandTimeout: 10000,
   requestTimeout: 15000,
+  chromeWebSecurity: false,
```

Comment:
I know that we likely need to disable `chromeWebSecurity` due to captcha requiring third-parties. But is there a way to isolate it to our captcha tests rather than all tests?

Reply:
I'm also not sure if this will work in ci e2es since this is only the dev env config. On second thought, if we are using the mock service now, do we need this flag?

Reply:
It already used in https://github.com/SAP/spartacus/blob/6ddc4263feaad8fb1d275ffa06cbcc9f03b815b5/projects/storefrontapp-e2e-cypress/cypress.config.ci.ts#L16 

For me, I need this change, or the server will reject our request as we have. "commerce-cloud-captcha-token" which need to configure to server's allow list.  See  https://help.sap.com/docs/INTERNAL_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/4605061113284c2c92cc558600140aaa.html?locale=en-US&state=DRAFT#loio4605061113284c2c92cc558600140aaa

Reply:
I have tried to find a solution to dynamic change this settings, looks no way  :(